tests/test_cases/test_cocotb/test_concurrency_primitives.py

```diff
@@ -11,68 +11,64 @@
 
 
 @cocotb.test()
-def test_yield_list_stale(dut):
-    """ Test that a trigger yielded as part of a list can't cause a spurious wakeup """
+async def test_await_list_stale(dut):
+    async def test_unfired_first_triggers(dut):
```

Comment:
Original comment didn't have the test definition line in the suggestion.
```suggestion
async def test_unfired_first_triggers(dut):
```